### DISCUSSION TITLE
VectorDB Elasticsearch

### DISCUSSION THREAD

**Discussion by zanderjiang** [microsoft/promptflow#3451] [Q&A]
The latest 0.2.10 version of vectordb seems to support Elasticsearch, but there doesn't seem to be any documentation on the topic. Anyone incorporated Elasticsearch index in promptflow yet?

**Comment by brynn-code** [comment]
@dans-msft @Adarsh-Ramanathan could you please take a look at this discussion? Thanks.

**Reply by e-straight** [reply]
@Jackxu9946
